runtime/autoload/man.vim

```diff
@@ -1,137 +1,211 @@
-let s:man_tag_depth = 0
-let s:man_sect_arg = ''
-let s:man_find_arg = '-w'
-
-try
-  if !has('win32') && $OSTYPE !~? 'cygwin\|linux' && system('uname -s') =~? 'SunOS' && system('uname -r') =~? '^5'
-    let s:man_sect_arg = '-s'
-    let s:man_find_arg = '-l'
```

Comment:
When I was working on neoman, since I couldn't test any of this, I decided to remove for simplicity's sake, but I'll add it back in.